File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1117.c
#include <stdio.h>

int main(){
    double n1=-1, n2, nota;
    while(1){
        scanf("%lf", &nota);
        if(nota>=0 && nota<=10){
            if(n1<0){
                n1=nota;
            } else{
                n2=nota;
                break;
            }
        } else{
            puts("nota invalida");
        }
    }
    printf("media = %.2lf\n", (n1+n2)/2);

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1117.rb
n1=-1;
while(true) do
    nota = gets.to_f
    if(nota>=0 && nota<=10) then
        if(n1<0) then
            n1=nota
        else
            n2=nota
            break
        end
    else
        puts("nota invalida")
    end
end
puts("media = %0.02f" % ((n1+n2)/2).round(2))